### ISSUE TITLE
Could rainbowstream track hashtags?

### ISSUE THREAD

**Issue by h8h** [DTVD/rainbowstream#103]
[switch public #AKB48 will switch current stream to public stream and track keyword AKB48](
 http://rainbowstream.readthedocs.org/en/latest/#the-interactive-mode)

Could I track keywords with a hashtag too? I.e. switch public #AKB48 will switch current stream to public stream and track keyword **#**AKB48.

ATM it doesn't matter if I type
* switch public **#**AKB48
* switch public AKB48

cheers

**Comment by DTVD** [comment]
I just checked the code, at this time RS will strip out any "#" on a keywords, so as you said, it will return the same stream with `switch public #AKB48` either `switch public AKB48`

I will take a further look if I can support keyword with "#" or why I did that stripping out before. Thanks !

**Comment by h8h** [comment]
Thanks for looking into this.

**Comment by pjoubert-** [comment]
Just for information, I tried to add a second # to the hashtag and only one # was stripped, letting me to follow hashtag's stream.

**Comment by h8h** [comment]
Yep thats possible though, but I hope thats not the solution :-)
